packages/setup-ocaml/src/opam.ts

```diff
@@ -27,8 +28,8 @@ import {
 } from "./system";
 import { getCygwinVersion } from "./win32";
 
-async function getLatestOpamRelease() {
-  const semverRange = "<2.2.0";
+export async function getLatestOpamRelease() {
+  const semverRange = ALLOW_PRELEASE_OPAM ? "<2.3.0" : "<2.2.0";
```

Comment:
We could use `*` here instead of `<2.3.0` 🤷‍♂️

Reply:
Could be. I don't think it hurts to keep `2.3.0` though, whenever a new version is released, both numbers should be updated but they are really close to each other in the code, so it'd hard to miss the update to the prerelease version.